benchmarks/bge_m3/src/scripts/export_model.py

```
@@ -0,0 +1,76 @@
+    )
+    parser.add_argument(
+        "--dtype",
+        default="float32",
+        help="Data type: float32, float16, int8",
+    )
+    args = parser.parse_args()
+
+    try:
+        export_model(args.model, args.output, args.optimize, args.dtype)
+        return 0
+    except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)